monitors/test/CRMonitor.js

```diff
@@ -54,7 +54,7 @@ contract("CRMonitor.js", function(accounts) {
     collateralToken = await Token.new("Dai Stable coin", "Dai", 18, { from: tokenSponsor });
 
     identifierWhitelist = await IdentifierWhitelist.deployed();
-    await identifierWhitelist.addSupportedIdentifier(web3.utils.utf8ToHex("UMATEST"));
+    await identifierWhitelist.addSupportedIdentifier(web3.utils.utf8ToHex("ETHBTC"));
```

Comment:
Update to this test file is unrelated to the core PRs changes. This change was made to improve this monitors test coverage and quality